packages/@webex/plugin-meetings/src/constants.ts

```diff
@@ -492,6 +500,30 @@ export const ERROR_DICTIONARY = {
     MESSAGE: 'No meeting info found for the meeting',
     CODE: 10,
   },
+  RECLAIM_HOST_NOT_SUPPORTED: {
```

Comment:
Nipick: May be we can match all the prefixes and call it RECLAIM_HOST_ROLE_NOT_SUPPORTED